### DISCUSSION TITLE
blog post tags on the main nav page

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#4393] [Ideas]
Hi all,
it looks like tags are only displayed when a user opens the post page. I think it is also good (and quite common) to show the tags on the main page somewhere in the post meta

**Comment by squidfunk** [comment]
This could be added as an option, but I'm unsure where to put them. Above the header as on the page? Below? I've left them out because when I included them, it looked very cluttered. Your current options are: you can override the theme and add them to the `partials/post.html` at the location you wish. I'll see whether we can offer some options on what to do here.

**Reply by squidfunk** [reply]
In fact, you can also open a feature request so we can check whether other users are wishing for the same functionality. If this is something very common, we should definitely include it. Also, I'd be interested in how other blogging solutions do it – building off learned behavior is often a good idea.

**Reply by hellt** [reply]
proposed via a feature request https://github.com/squidfunk/mkdocs-material/issues/4421
